crates/pop-cli/src/commands/install/frontend.rs

```diff
@@ -0,0 +1,162 @@
+// SPDX-License-Identifier: GPL-3.0
+
+use crate::{
+	cli::{self, traits::Confirm},
+	commands::install::run_external_script,
+	common::binary::SemanticVersion,
+};
+use anyhow::{Result, anyhow};
+use duct::cmd;
+use std::path::PathBuf;
+
+const MIN_NODE_VERSION: u8 = 20;
```

Comment:
I didn't update my node version the first time I run the command and got this warning:

<img width="1076" height="851" alt="Image" src="https://github.com/user-attachments/assets/54f6e463-d397-481b-be14-2ac0becd4a05" />

Maybe we should set the minimum to 24?